lib/jsonapi/utils/underscore.ex

```diff
@@ -56,14 +60,44 @@ defmodule JSONAPI.Utils.Underscore do
   end
 
   def underscore(value) when is_map(value) do
-    Enum.into(value, %{}, &underscore/1)
+    Enum.into(value, %{}, &underscore(&1))
```

Comment:
For my own edification, what is the purpose of this change?

Reply:
Nice catch—this is leftover from a previous version that added a parameter to the call. Now unnecessary